modules/overrides/user_configurable_overrides.go

```diff
@@ -195,6 +198,14 @@ func (o *userConfigurableOverridesManager) setTenantLimit(userID string, limits
 	}
 }
 
+func (o *userConfigurableOverridesManager) GetTenantIDs() []string {
+	var ids []string
+	for tenant := range o.getAllTenantLimits() {
```

Comment:
nit: Could use `maps.Keys(o.getAllTenantLimits)` (same for both implementations).